exosphere-shared/example-apps/crashing-service/application.yml

```diff
@@ -6,5 +6,7 @@ services:
   public:
     crasher:
       location: ./crasher
+      docker_image: exospheredev/test-service-that-crashes-on-startup
```

Comment:
Okay, makes sense as a first and most simple solution. Thanks for the explanation!